### ISSUE TITLE
Implement Login Page Layout & Routing

### ISSUE THREAD

**Issue by gokul472** [overture-stack/dms-ui#8]
Implement the basic login page layout per the mockups (without any API integration yet) and basic routing.

For routing, nextjs was previously chosen already as the framework to use.

Start with Google as the first implementation to get one login flow working before attempting the other providers.

**Screenshot:**

![image.png](https://images.zenhubusercontent.com/5ec563ba889df3e7a41572b3/83ba18f6-2bb1-445d-8226-00d84f1f923f)

**Mockups:**
Login Page: https://zpl.io/2GZdPRj

**Comment by b-f-chan** [comment]
Testing blocked waiting on #4 to get default metadata into Arranger, will carryover to Sprint 7

**Comment by b-f-chan** [comment]
Arranger setup complete, testing on QA & Staging unblocked.   Unblocking this ticket.

**Comment by b-f-chan** [comment]
As discussed with @anncatton, scope of this ticket is for the layout and routing.   Work to connect login button to the EGO endpoint and implement the actual login flow will be in separate ticket (see dms-ui#9).

**Testing on QA:**

Passed:

- Header logo correctly displayed
- 5 login options correctly displayed
- Overture image & page layout correctly displayed
- Footer with doc links correctly displayed

Questions/Issues:

1. Although login/logout logic is not in scope of this ticket, why is only Google and ORCID enabled?  Because only those 2 filters are configured/enabled for this deployment?
2. Although login/logout logic is not in scope of this ticket, why does the user show as logged in on the login page?  The "Hello <User>" control should never show on the Login page correct?  Only other pages after login (e.g. Repository)?
3. Minor nitpick: Font weight is bold for "Please choose one of the following...", but is not in Zeplin.  Do we care?  Which do we prefer?
4. Minor issue: The Overture image on the right side is not responsive, as you shrink the browser window the image overlaps the login buttons.  To be fixed in separate ticket.

Other Notes:

- Doc links (Help, DMS) present, but content not yet available, to be produced in separate tickets

**Comment by b-f-chan** [comment]
Will move to Preview column once Arranger config fixed in Staging and we can do proper release to Staging for DMS-UI.

**Comment by b-f-chan** [comment]
Additional feedback from @kcullion, we will handle these in a single feedback ticket as most are minor:

- Add hover state for login buttons
- Overture logo in footer should link to overture.bio
- Login instructions should not be bold

**Comment by kcullion** [comment]
Thanks @b-f-chan ..just to add for the image - I think on mobile or smaller screens we can hide this illustration/green background altogether. So right when the layout breaks, hide the image

**Comment by rosibaj** [comment]
lgtm on staging 
feedback is captured in linked ticket 
closing as feedback will be addressed in future tickets. 

Thanks @anncatton !
